### PR TITLE
Reduce cover size & update max cover period to 365 days

### PR DESCRIPTION
**Pull request by roxdanila**
## Context

Latest PRs increased Cover size past limit again.

## Changes proposed in this pull request

- make more constants private
- remove IC unused dependency
- made max cover period 365 days instead of 364 (with @shark0der 's blessing)

## Test plan
Ran existing tests

## Checklist

- [x] Rebased the base branch
- [x] Attached corresponding Github issue
- [x] Prefixed the name with the type of change (i.e. feat, chore, test)
- [x] Performed a self-review of my own code
- [x] Followed the style guidelines of this project
- [ ] Made corresponding changes to the documentation
- [x] Didn't generate new warnings
- [x] Didn't generate failures on existing tests
- [ ] Added tests that prove my fix is effective or that my feature works

## Review

When reviewing a PR, please indicate intention in comments using the following emojis:
* :cake: = Nice to have but not essential.
* :bulb: = Suggestion or a comment based on personal opinion
* :hammer: = I believe this should be changed.
* :thinking: = I don’t understand something, do you mind giving me more context?
* :rocket: = Feedback

### CODE REVIEW

File: contracts/modules/cover/Cover.sol [hunk 1]
```diff
@@ -63,9 +62,9 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
   uint private constant GLOBAL_CAPACITY_DENOMINATOR = 10_000;
   uint private constant REWARD_DENOMINATOR = 10_000;
 
-  uint public constant MAX_COVER_PERIOD = 364 days;
+  uint private constant MAX_COVER_PERIOD = 365 days;
```
Comment (danoctavian) [inline contracts/modules/cover/Cover.sol:65]: why is there a value increase here?

Doesn't this affect the bucket size computation with 91 * 4 = 364?

Comment (shark0der) [reply contracts/modules/cover/Cover.sol:65]: Most of the time we'll be mid-tranche (not bucket). This means that we'll use 4 tranches for a 1-year cover anyway so the 364 days max only helps in very few cases.

Comment (danoctavian) [reply contracts/modules/cover/Cover.sol:65]: sorry meant to say tranche, ok understood

**Review by danoctavian — APPROVED** [review]
LGTM

**Pull request merged**